CHANGELOG.md

```diff
@@ -8,10 +8,22 @@ Useful links: Docsy [releases][] & [tags][]. Jump to the [latest][] release.
 
 For a list of issues targeted for the next release, see the [22Q2][] milestone.
 
-## [0.5.0][] - next planned release (unpublished yet)
+## 0.6.0 - next planned release (unpublished yet)
+
+
+**New**:
+
+
+**Breaking changes**:
+
+
+**Other changes**:
+
+
+## [0.5.1][]
 
 For the full list of the changes found in this release, see the [release
-notes][0.5.0]. **BREAKING CHANGES** are documented below.
+notes][0.5.1]. **BREAKING CHANGES** are documented below.
```

Comment:
You'll also need to update the markdown link definition on line 166 below, from 

```md
[0.5.0]: https://github.com/google/docsy/releases/v0.5.0
```

to

```md
[0.5.1]: https://github.com/google/docsy/releases/v0.5.1
```

Reply:
Good catch, thank you!